governance/API-Ref-Guidelines-Update.md

```diff
@@ -0,0 +1,508 @@
+# API Ref Guidelines <!-- omit in toc -->
+
+## Table of Contents <!-- omit in toc -->
+
+- [Scope](#scope)
+- [Who is the customer?](#who-is-the-customer)
+- [How much documentation should I add?](#how-much-documentation-should-i-add)
+- [Can I exclude code from being included in the documentation?](#can-i-exclude-code-from-being-included-in-the-documentation)
+- [How should I format my code comments?](#how-should-i-format-my-code-comments)
+  - [General Style](#general-style)
+  - [Classes and Structures](#classes-and-structures)
+  - [Member Data](#member-data)
+  - [Functions and Function Parameters](#functions-and-function-parameters)
+  - [Macros](#macros)
+  - [Enumerations](#enumerations)
+  - [Namespaces](#namespaces)
+  - [Header files](#header-files)
+  - [Deprecations](#deprecations)
+  - [Complete Example](#complete-example)
+- [Extras](#extras)
+  - [Linking to other APIs](#linking-to-other-apis)
+  - [Linking to Other Guides and Sites](#linking-to-other-guides-and-sites)
+  - [See Also](#see-also)
+  - [Notes](#notes)
+  - [Code Snippets](#code-snippets)
+  - [Section Comments](#section-comments)
+- [Excluding code from the documentation](#excluding-code-from-the-documentation)
+- [Exceptions](#exceptions)
+- [As an SDE, can I build the documentation to see the final result?](#as-an-sde-can-i-build-the-documentation-to-see-the-final-result)
+
+## Scope
+
+This page describes how to comment O3DE's C++ code to produce consistent, high-quality documentation for O3DE’s public-facing C++ API Reference Guide. The intended audiences of _this page_ are O3DE SDE's and programming writers.
+
+We list the minimum code elements to document, describe how to add the documentation, and provide examples of documenting different types of elements. Descriptions of how to build and publish the API Reference Guide are out of scope of this document.
+
+## Who is the customer?
+
+An API Reference consists of two major parts
+
+1. The actual header and cpp files that are shipped as part of O3DE, viewed by SDEs in their IDE as they work on the code.
```

Comment:
SDE might be too narrow a term, software engineer might be better, or even just something more generic.

Reply:
Good callout, I've just gone with `developers`